tools/KoreBuild.Console/Commands/DisableSkipStrongNameCommand.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Win32;
+using SkipStrongNames;
+
+namespace KoreBuild.Console.Commands
+{
+    internal class DisableSkipStrongNameCommand : BaseSkipStrongNameCommand
```

Comment:
Can we make this a task? If it works, it's much nicer than spawning a process and reading xml files.